config/owasp/suppressions.xml

```diff
@@ -16,12 +16,11 @@
 
   <!--Please add all the temporary suppression under the below section -->
   <suppress>
-    <notes><![CDATA[
-      file name: tomcat-embed-core-9.0.58.jar
-      ]]>
-      https://tools.hmcts.net/jira/browse/CCD-3270
-    </notes>
+    <notes>
+      CVE-2022-29885 refer https://tools.hmcts.net/jira/browse/CCD-3270
+      CVE-2022-34305 refer https://tools.hmcts.net/jira/browse/CCD-3433
     <cve>CVE-2022-29885</cve>
```

Comment:
@dinesh1patel - notes end tag seems to be missing

Reply:
@rajaram-nagarajan done